### PR TITLE
Add time to patcher cli

### PR DESCRIPTION
**Pull request by dyceron**
Helpful information

### CODE REVIEW

File: src/open_samus_returns_rando/cli.py [hunk 1]
```diff
@@ -60,9 +61,12 @@ def main() -> None:
...
 
+    start = time.time()
     samus_returns_patcher.patch_extracted(
         args.input_path,
         args.input_exheader,
         args.output_path,
         configuration,
     )
+    end = time.time()
+    print(f"Patcher took {end - start:.03f} seconds")
```
Comment (Miepee) [inline src/open_samus_returns_rando/cli.py:72]: Methinks this should not be in release builds

Comment (dyceron) [reply src/open_samus_returns_rando/cli.py:72]: It exists for odr so 🤷‍♂️

Comment (henriquegemignani) [reply src/open_samus_returns_rando/cli.py:72]: Randovania doesn't use this function and it's convenient when running manually.

Comment (Miepee) [reply src/open_samus_returns_rando/cli.py:72]: It still shows up in randovania builds via the cmd console that pops up on windows tho, no?

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**